src/ngraph/runtime/gpu/gpu_primitive_emitter.cpp

```diff
@@ -14,10 +14,8 @@
 // limitations under the License.
 //*****************************************************************************
 
-#include <limits>
-
-#include "ngraph/runtime/gpu/cudnn_emitter.hpp"
 #include "ngraph/runtime/gpu/gpu_primitive_emitter.hpp"
+#include <limits>
```

Comment:
<> includes go before ""

Reply:
I tried pushing it above, but due to some reason, for this case atlst, clang format keeps putting it below

Reply:
It does this because the header file for this cpp file, gpu_primitive_emitter.hpp in this example, always goes first in the list. If you put a blank line between <> and "" includes then all is well.